analyzers/tests/SonarAnalyzer.UnitTest/Constants.cs

```diff
@@ -23,6 +23,8 @@ namespace SonarAnalyzer.UnitTest
     internal static class Constants
     {
         public const string NuGetLatestVersion = "LATEST";
+        public const string DotNet7Preview = "7.0.0-rc.1.22426.10";
+        public const string DotNet7PreviewAzureAppServices = "7.0.0-rc.1.22427.2";
```

Comment:
`AzureAppServices` has a slightly different version. This should be removed after the official release.